pkg/daemon/daemon.go

```diff
@@ -162,30 +165,38 @@ func (p *ProcessManager) UpdateSynceConfig(config *synce.Relations) {
 
 }
 
+type tBCProcessAttributes struct {
+	controlledPortsConfigFile string
+	trIfaceName               string // Time receiver interface name for T-BC clock monitoring
+}
+
 type ptpProcess struct {
-	name                string
-	ifaces              config.IFaces
-	processSocketPath   string
-	processConfigPath   string
-	configName          string
-	messageTag          string
-	eventCh             chan event.EventChannel
-	exitCh              chan bool
-	execMutex           sync.Mutex
-	stopped             bool
-	logFilters          []*logfilter.LogFilter // List of filters to apply to logs
-	cmd                 *exec.Cmd
-	depProcess          []process // these are list of dependent process which needs to be started/stopped if the parent process is starts/stops
-	nodeProfile         ptpv1.PtpProfile
-	parentClockClass    float64
-	pmcCheck            bool
-	clockType           event.ClockType
-	ptpClockThreshold   *ptpv1.PtpClockThreshold
-	haProfile           map[string][]string // stores list of interface name for each profile
-	syncERelations      *synce.Relations
-	c                   *net.Conn
-	hasCollectedMetrics bool
-	trIfaceName         string // Time receiver interface name for T-BC clock monitoring
+	name                  string
+	ifaces                config.IFaces
+	processSocketPath     string
+	processConfigPath     string
+	configName            string
+	messageTag            string
+	eventCh               chan event.EventChannel
+	exitCh                chan bool
+	execMutex             sync.Mutex
+	stopped               bool
+	logFilters            []*logfilter.LogFilter // List of filters to apply to logs
+	cmd                   *exec.Cmd
+	depProcess            []process // these are list of dependent process which needs to be started/stopped if the parent process is starts/stops
+	nodeProfile           ptpv1.PtpProfile
+	ParentDataSet         *protocol.ParentDataSet
+	TimePropertiesDataSet *protocol.TimePropertiesDS
+	CurrentDS             *protocol.CurrentDS
```

Comment:
can I have inline comments for this , what they are used for ? 
ParentDataSet         *protocol.ParentDataSet
TimePropertiesDataSet *protocol.TimePropertiesDS
CurrentDS             *protocol.CurrentDS

Reply:
Moved them to the T-BC structure and added comments